make/modules/jdk.jpackage/Lib.gmk

```diff
@@ -129,7 +129,7 @@ ifeq ($(call isTargetOs, windows), true)
       CXXFLAGS := $(CXXFLAGS_JDKLIB) $(JPACKAGE_CXXFLAGS_windows), \
       LDFLAGS := $(LDFLAGS_JDKLIB) $(LDFLAGS_CXX_JDK) \
           $(call SET_SHARED_LIBRARY_ORIGIN), \
-      LIBS := $(LIBCXX) advapi32.lib ole32.lib msi.lib user32.lib \
+      LIBS := advapi32.lib ole32.lib msi.lib user32.lib \
```

Comment:
Should this (and the others below) be using LINK_TYPE:=C++ or was $(LIBCXX) never actually needed on Windows?

Reply:
$(LIBCXX) is not needed on Windows. It is used only on gcc, where it expands to flags needed to link statically with the C++ standard libraries, if static linking with the standard libs is selected.

There's overall a lot of messiness with the flags, with setting flags that is pointless, or omitting to set flags that we should (due to copy/paste once again). These initial PRs I've posted try to keep the same flags for all libraries. The fun (or "fun") part comes after that, when I'll try to modify the flags to make more sense.